jupyterhub/handlers/base.py

```diff
@@ -83,6 +83,7 @@ async def prepare(self):
         except Exception:
             self.log.exception("Failed to get current user")
             self._jupyterhub_user = None
+            self.scopes = []
```

Comment:
Let's move this up to an unconditional initial value, so we only override it when we have some scopes to set